book/graphs/other/cig01.qmd

```diff
@@ -147,13 +151,16 @@ ggplot(
 
 The corresponding table is simply obtained using the `rtables` framework:
 
-```{r cig01_6}
+```{r table6, test = list(table_v6 = "table")}
 lyt <- basic_table() %>%
   split_cols_by(var = "ARMCD") %>%
   summarize_vars(vars = "AVAL", .stats = c("mean_sd", "median"))
-build_table(lyt = lyt, df = adlb)
+table <- build_table(lyt = lyt, df = adlb)
+table
```

Comment:
There can be markdown tables present inside the quarto docs inside the graphs directory. So the `test-graph-snaps.R` should handle this.